next.dynamic.mjs

```diff
@@ -212,6 +212,21 @@ const getDynamicRouter = async () => {
       path
     );
 
+    const match = path.match(/blog\/(release|vulnerability)/);
+
+    if (match) {
+      const category = match[1];
+      const currentFile = siteConfig.rssFeeds.find(
+        item => item.category === category
+      )?.file;
+      // Dynamically construct the XML path for blog/release and blog/vulnerability
+      pageMetadata.alternates.types['application/rss+xml'] =
+        `${baseUrlAndPath}/en/feed/${currentFile}`;
+    } else {
+      pageMetadata.alternates.types['application/rss+xml'] =
+        `${baseUrlAndPath}/en/feed/blog.xml`;
```

Comment:
Should these be constructed using `getUrlForPathname`, so the locale changes correctly?

Reply:
I believe currently we are always defaulting to en/feed/blog.xml this have to change for future as well

Reply:
Done